team/bundles/org.eclipse.core.pki/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,35 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %PLUGIN_NAME
+Bundle-Localization: plugin
+Bundle-SymbolicName: org.eclipse.core.pki; singleton:=true
+Require-Bundle: org.eclipse.equinox.security;bundle-version="[1.0.0,2.0.0)",
+ org.eclipse.equinox.common;bundle-version="3.12.0",
+ org.eclipse.equinox.preferences;bundle-version="3.2.200",
+ org.eclipse.osgi;bundle-version="3.4.0",
+ org.eclipse.equinox.registry;bundle-version="3.4.0",
+ org.eclipse.ui;bundle-version="3.203.100",
+ org.eclipse.ui.workbench,
+ org.eclipse.core.runtime,
+ org.eclipse.core.net
+Bundle-Version: 1.0.0
+Export-Package: org.eclipse.core.pki,
+ org.eclipse.pki.auth,
+ org.eclipse.pki.dialog,
+ org.eclipse.pki.exception,
+ org.eclipse.pki.interfaces,
+ org.eclipse.pki.jsse,
+ org.eclipse.pki.pkcs,
+ org.eclipse.pki.pkiselection,
+ org.eclipse.pki.preferences,
+ org.eclipse.pki.ui,
+ org.eclipse.pki.util,
+ org.eclipse.pki.wizard,
+ target.classes.org.eclipse.core.pki
```

Comment:
This line is clearly wrong. The `target.classes` is not a package, it's a mistake coming from erroneous project configuration.

Reply:
remove target